docs/production-deployment/cloud/get-started/billing-and-cost.mdx

```diff
@@ -113,16 +116,15 @@ If your organization separates work by Namespace—for development, production,
 :::note Cost Breakdown Limitations
 
 Namespace cost details are not available for "last 90 days" or "last 120 days".
-Cost breakdowns are estimated as the total invoices prorated by usage across namespaces to account for Actions/Storage
-included in your Temporal plan.
 
-:::
+Cost breakdowns distribute the total usage cost to namespaces proportionally based on their metered usage. The proration
+reflects your effective price, factoring in included Actions/Storage and tiered pricing rates in your Temporal plan.
 
-You may download your Invoices prior to this calendar month by clicking the download icon by the date.
+:::
 
 ## Plans {#plans}
 
-Account owners and Finance Admin can access their Temporal Plan information on the plans page.
+Account Owners and Finance Admins can access their Temporal Plan information on the plans page.
```

Comment:
feel free to modify but i updated these to make the text casing and plurality consistent